pyrefinebio/http.py

```diff
@@ -79,12 +79,56 @@ def put_by_endpoint(endpoint, payload=None):
     url = config.base_url + endpoint + "/"
     return put(url, payload=payload)
 
-def download_file(url, path):
+def download_file(url, path, prompt):
+    user_path = os.path.expanduser(path)
+    full_path = os.path.abspath(user_path)
+
+    if os.path.isdir(full_path):
+        full_path = os.path.join(full_path, "refinebio-download.zip")
+
     with requests.get(url, stream=True) as res:
-        with open(path, "wb") as f:
+
+        if prompt:
+            total_size_in_bytes = int(res.headers.get("content-length", -1))
+            message = None
+
+            if total_size_in_bytes == -1:
+                message = (
+                    "Could not get the size for the file you are tying to download. "
+                    "Would you still like to download it? (y/n)"
+                )
+            if total_size_in_bytes > 1000000000:
+                message = (
+                    "The file you are trying to download is bigger than 1GB. "
+                    "Would you still like to download it? (y/n)"
```

Comment:
```suggestion
                    "Would you still like to download it? (y/N)"
```
This is a convention which indicates that if nothing is entered, `No` will be the default choice.